cmd/pocketd/cmd/commands.go

```diff
@@ -58,6 +58,70 @@ func initRootCmd(
 func addModuleInitFlags(startCmd *cobra.Command) {
 	//nolint:staticcheck // SA1019 TODO_TECHDEBT: remove deprecated code.
 	crisis.AddModuleInitFlags(startCmd)
+
+	// Add additional CometBFT P2P flags that aren't included by default
+	startCmd.Flags().Int("p2p.max-num-inbound-peers", 40, "Maximum number of inbound peers")
```

Comment:
Can you look at the changes in #1444 and follow the patterns we introduced there?